runtime/src/accounts/mod.rs

```diff
@@ -52,7 +51,6 @@ pub(super) fn load_accounts(
     ancestors: &Ancestors,
     txs: &[SanitizedTransaction],
     lock_results: Vec<TransactionCheckResult>,
-    hash_queue: &BlockhashQueue,
```

Comment:
`hash_queue` was used to resolve `lamports_per_signature`, which is no longer needed.